packages/cli/package.json

```diff
@@ -80,5 +80,8 @@
     "tsup": "^6.7.0",
     "tsx": "^3.12.6",
     "vitest": "0.34.6"
+  },
+  "peerDependencies": {
+    "@aws-sdk/client-kms": "^3.556.0"
```

Comment:
I think this needs to be an actual dependency. I deleted my earlier comment after realizing this was in the CLI package, not a package end-users would depend on.

Reply:
I think we also need to install asn1 here